src/interactive.c

```diff
@@ -15,7 +15,7 @@ max_move_scale(double pos_cursor, double pos_current,
 }
 
 void
-interactive_begin(struct view *view, enum input_mode mode, uint32_t edges)
+interactive_begin(struct view *view, enum input_mode mode, enum wlr_edges edges)
```

Comment:
`interactive_begin()` is passed a `uint32_t` or literal `0` in the places I see.  It would be nice to be consistent -- either stick with `uint32_t` everywhere or switch over to `wlr_edges` everywhere.

Reply:
Yes you have a point there. I think I'll try to use the enum everywhere, including `WLR_EDGE_NONE` instead of `0`.
Will split that uint32_t -> enum stuff up into an own commit.